ee-feature-pack/common/src/main/resources/content/bin/jconsole.bat

```diff
@@ -46,6 +46,12 @@ rem Setup The Classpath
 
 set "CLASSPATH=%JAVA_HOME%\lib\jconsole.jar"
 set "CLASSPATH=%CLASSPATH%;%JAVA_HOME%\lib\tools.jar"
+
+if not exist "%JBOSS_HOME%\bin\client\jboss-cli-client.jar" (
+    echo   WARNING: Jar not found: %JBOSS_HOME%\bin\client\jboss-cli-client.jar
+    echo   WARNING: If this jar is missing, jconsole will fail to connect to Wildfly.
```

Comment:
s/to WildFly/to the server/g

I don't want to have to have add these to the EAP rebranding commits in https://github.com/jbossas/wildfly-to-product.

Reply:
@lvydra Also, if anything, we would stylize it as "WildFly" (just saying for future reference).